tests/on-device/config-flake-h-m.bats

```diff
@@ -3,10 +3,7 @@
 load lib
 
 @test 'flake + h-m + #134 overlays case work' {
-  # start from a known baseline
   switch_to_default_config
-  assert_command vi
```

Comment:
If the test is testing "X was there, now it's not" or vice versa, but the first check movest to some other file, it's prone to changing with time and then suddenly we can find ourselves not testing that activation changed something.

Reply:
Okay understood. So my idea would then be: move switch to default without assertions to setup because it is just setup. Add all for this specific test case relevant assertions at the top of the test and remove all other ones at the top that do not bring value for this specific test. I will mark these assertions with a comment.